### PR TITLE
Added tap-zoom to bars in Joey's reports. Closes #425

### LINKED ISSUE #425: Tap-to-zoom for barchart month view
Bar tap @ month view zooms three closest rows to full(normal) height and makes durations visible.
Epic #178

### PR DESCRIPTION
**Pull request by monday8am**
Added methods to make zoom tapping on bars. Fixes #425

### CODE REVIEW

File: Joey/Joey.csproj [hunk 1]
```diff
@@ -71,6 +71,8 @@
     <Reference Include="GooglePlayServicesLib">
       <HintPath>..\packages\Xamarin.GooglePlayServices.19.0.0.1\lib\MonoAndroid23\GooglePlayServicesLib.dll</HintPath>
     </Reference>
+    <Reference Include="System.ComponentModel.Composition" />
+    <Reference Include="Mono.Android.Export" />
```
Comment (roosmaa) [inline Joey/Joey.csproj]: Why those references?

Comment (monday8am) [inline Joey/Joey.csproj]: the first I'm not complete sure, the second is related with [Export]  attribute.

File: Joey/UI/Views/BarChart.cs [hunk 2]
```diff
@@ -307,6 +314,195 @@ protected override void OnLayout (bool changed, int l, int t, int r, int b)
...
+                };
+
+                var valueFadeIn = ObjectAnimator.OfFloat (row.ValueTextView, "alpha", row.ValueTextView.Alpha, 0f).SetDuration (300);
+                valueFadeIn.AnimationEnd += (sender, e) => {
+                    if ( row.ValueTextView.Alpha == 0f) {
+                        row.ValueTextView.Visibility = ViewStates.Gone;
+                    }
+                };
+
+                var barHeight = ObjectAnimator.OfInt ( row, "zoomedHeight", row.BarView.Height, rowHeight).SetDuration (300);
```
Comment (roosmaa) [inline Joey/UI/Views/BarChart.cs]: I think you should be able to use regular C# properties as well (Xamarin does some magic), so no need to [Export] them to the Java world.
Ie `ZoomedHeight` property and `"ZoomedHeight"` string argument to animator.

Comment (monday8am) [inline Joey/UI/Views/BarChart.cs]: I tried but I received a "[PropertyValuesHolder] Method setXXX() with type int not found on target class class YYY" message. In other words, the compiler don't find the property.

**Comment by roosmaa** [comment]
Superseded by #472

**Pull request closed without merging**